.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Since the project is typescript, I don't think this eslint config is doing anything. I think you need to get this setup: https://typescript-eslint.io/getting-started

Reply:
It's for test files. I make tests in JavaScript because it's not type strict and I don't want to use many `any` 😄

Reply:
... we should be linting src too tho, right? I mean, why wouldn't we...

Reply:
Gotcha

Reply:
And if this is the eslint config for the test files, can't the eslint config can go into the `tests` directory?